frontend/src/components/Table/Table.scss

```diff
@@ -74,6 +78,26 @@
       justify-content: flex-start;
     }
     .filter-wrapper {
+      div {
+        div {
+          display: flex;
+          .form-group {
+            width: 100%;
+            .rbt {
+              width: 100%;
+              div {
+                input {
+                  width: 100% !important;
```

Comment:
can we restructure something so that this is not necessary? the width 100 -> width 100 -> width 100 important seems not great.